src/components/dialogmote/referat/Referat.tsx

```diff
@@ -368,29 +453,37 @@ const Referat = ({
             {mellomlagreReferat.isError && (
               <SkjemaInnsendingFeil error={mellomlagreReferat.error} />
             )}
-            {submitFailed && harIkkeUtbedretFeil && (
-              <SkjemaFeiloppsummering errors={errors} />
-            )}
             {mellomlagreReferat.isSuccess && uendretSidenMellomlagring && (
               <Alert variant="success" size="small">
                 {savedReferatText(lastSavedTime.toDate())}
               </Alert>
             )}
-            <ReferatButtons
-              pageTitle={pageTitle}
-              onSaveClick={() => handleLagreClick(values)}
-              onSendClick={handleSendClick}
-              showSaveSpinner={mellomlagreReferat.isPending}
-              showSendSpinner={
-                ferdigstillDialogmote.isPending || endreReferat.isPending
-              }
-            />
-            <ForhandsvisningModal
-              contentLabel={texts.forhandsvisningContentLabel}
-              isOpen={displayReferatPreview}
-              handleClose={() => setDisplayReferatPreview(false)}
-              getDocumentComponents={() => getReferatDocument(values)}
-            />
+            <div className="flex gap-4 pt-12">
+              <Button
```

Comment:
Enklere å ha buttons direkte her i stedet for at de er "skjult" i `<ReferatButtons>`